thorlcr/activities/selectnth/thselectnth.cpp

```diff
@@ -35,7 +35,7 @@ class CSelectNthActivityMaster : public CMasterActivity
 CActivityBase *createSelectNthActivityMaster(CMasterGraphElement *container)
 {
     if (container->queryLocalOrGrouped())
-        return new CMasterActivity(container);
+        return new CMasterActivity(container, spillingActivityStatistics);
```

Comment:
as per previous comments. The local version doesn't use a lookahead. The global does.